core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -43,6 +51,18 @@ trait BillingProfileManagerDAO {
     ec: ExecutionContext
   ): Future[Seq[RawlsBillingProject]]
 
+  def addProfilePolicyMember(billingProfileId: UUID,
+                             role: ProjectRole,
+                             memberEmail: String,
+                             ctx: RawlsRequestContext
+  ): Unit
```

Comment:
So this DAO has half of its methods returning Future's and half returning bare values. I think we should covert all of them to bare values for consistency. Thoughts?

Reply:
Yes, it should be consistent-- this is just me not understanding Scala. I will try to sort it out.

Reply:
I agree that bare values are the way to go: the futures are only useful if you're doing something while it's executing.

Reply:
I converted all of them except 2-- `getAllBillingProfiles` and `getHardcodedAzureBillingProject` both rely on `samDAO.userHasAction`, which returns a Future. However, once we can delete the alpha group gating that can be removed (and the hardcoded method is getting deleted).

Reply:
you could wait in the method
I don't think it's important though, since it's going away.